app/plugins/adp-backend/seedData/delivery_programme.js

```diff
@@ -60,7 +60,7 @@ const records = [
     description:
       'The Core Artificial Intelligence (AI) programme is a collection of projects looking at the benefits and use cases of Gen AI, Machine Learning, etc.',
     arms_length_body_id: 'core-defra',
-    delivery_programme_code: 'CAI',
+    delivery_programme_code: 'COREAI',
```

Comment:
On the word document @kbabigumira shared it was 'coreai', would it impact the project (coreai mcu) creation if we change it to 'crai'? @danny-may

Reply:
ah @kbabigumira  we have validation to be 4 characters...

Reply:
we either allow this exception.. but will it causes issues on edit?

Reply:
If we scaffold with that.. can we edit after the fact OK?

Reply:
On the call we said we should be able to edit the name without it impacting the services but it would be good to double check before I seed the data in

Reply:
ok could we check that?

Reply:
thanks, if so, happy for it to go in as is

Reply:
The code can be CAI. We need to ensure that the name is COREAI though.

Reply:
> On the word document @kbabigumira shared it was 'coreai', would it impact the project (coreai mcu) creation if we change it to 'crai'? @danny-may

I reckon it uses the name not the programme code for the programme name. So it is okay to leave code as is. But change the name to use COREAI

Reply:
@kbabigumira If we use 'CAI' as the delivery code then the project name would be 'cai coreai mcu' as it uses the code as a prefix for the project name. @danny-may please correct me if I'm wrong

Reply:
I think we've agreed to use the 4 digit code, and accept that core ai will need to be scaffolded. is this fair summary?

Reply:
Okay I'll go ahead and change it to 'CRAI' @DaRoszko @kbabigumira